source/api/presentation/3.0/index.md

```diff
@@ -240,6 +240,19 @@ The value _MUST_ be a JSON array, with each item in the array being a JSON objec
 {"thumbnail": [{"id": "https://example.org/img/thumb.jpg", "type": "Image"}]}
 ```
 
+##### posterCanvas
+A single Canvas providing content that represents this resource, such as image to show while a duration-only Canvas is playing audio, or images, text and sound shown for video content before the user initiates playback. Whereas a client might use `thumbnail` for navigation or summary views of multiple resources, a client can choose to show content from `posterCanvas` as part of the presentation of this resource without requiring the content of `posterCanvas` to be part of this resource. The poster Canvas can have different dimensions to the Canvas(es) of this resource.
+
+  * A Collection _MAY_ have exactly one `posterCanvas`.
+  * A Manifest _MAY_ have exactly one `posterCanvas`.
+  * A Sequence _MAY_ have exactly one `posterCanvas`.
+  * A Canvas _MAY_ have exactly one `posterCanvas`.
+  * A Range _MAY_ have exactly one `posterCanvas`.
+  * Other resource types _MUST NOT_ have a poster canvas.
+
+Publishers should not assume that a client will display the content of the poster Canvas to the user. 
```

Comment:
Could add here a comment along the lines of @azaroth42's https://github.com/IIIF/api/pull/1356#issuecomment-356680303

Reply:
posterCanvas for Range should be shown in preference to posterCanvas for Manifest?